packages/generated-files-bot/package.json

```diff
@@ -22,7 +22,7 @@
     "start": "probot run ./build/src/generated-files-bot.js",
     "start:local": "node ./build/src/local.js",
     "pretest": "npm run compile",
-    "test": "cross-env LOG_LEVEL=fatal c8 mocha --exit build/test",
+    "test": "cross-env LOG_LEVEL=fatal c8 mocha --exit build/test --timeout 10000",
```

Comment:
I am generally not a fan of the longer timeout, but I will read ahead

Reply:
Talking with @bcoe, it sounds like octokit is retrying with backoff until a certain point limit. The test takes about 3 seconds to pass which was higher than the 2s timeout.

Reply:
Opened https://github.com/googleapis/repo-automation-bots/issues/1078 to help address this. I like the current typing in the current implementation (no casting to `any`), but the retry on the 404 means we might need to bump the test timeouts.